src/components/FeedbackWidget.tsx

```diff
@@ -33,6 +36,17 @@ const FixedDot = styled(NakedButton)<{
     bottom: ${({ bottomOffset }) => 1 + bottomOffset}rem;
     margin-top: 150vh;
   }
+  @media (min-width: ${({ theme }) => theme.breakpoints.l}) {
+    width: ${({ isExpanded }) => (isExpanded ? "15rem" : "3rem")};
+    border-radius: ${({ isExpanded }) => (isExpanded ? "50px" : "50%")};
+    #feedback-wrapper {
+      width: ${({ isExpanded }) => (isExpanded ? "13.5rem" : "3rem")};
+      position: ${({ isExpanded }) => (isExpanded ? "absolute" : "flex")};
+    }
+    #expanded-prompt {
+      display: ${({ isExpanded }) => (isExpanded ? "flex" : "none")};
```

Comment:
I don't love the idea of using ids to achieve this. Why can't we implement this logic using style props?

Can't we do this like
```tsx
<Text ... width={{ base: "3rem", lg: isExpanded ? "13.5rem" : "3rem" }}>
  <Box
    as={Translation}
    id="feedback-card-prompt-page"
    display={{ base: "flex", lg: isExpanded ? "flex" : "none" }}
  />
</Text>
```

Reply:
@pettinarip thanks for the feedback and for the improvements you've made 🙂